src/chainstate/stacks/db/blocks.rs

```diff
@@ -3738,6 +3739,54 @@ impl StacksChainState {
         all_receipts
     }
 
+    /// Process any STX transfer bitcoin operations
+    ///  that haven't been processed in this Stacks fork yet.
+    pub fn process_transfer_ops(
+        clarity_tx: &mut ClarityTx,
+        operations: Vec<TransferStxOp>,
+    ) -> Vec<StacksTransactionReceipt> {
+        let mut all_receipts = vec![];
+        // NOTE: do _not_ return from this function without replacing this original tracker
```

Comment:
Could this be enforced by the compiler?  Maybe you could encapsulate the "body" of this method in a closure and run it, so any internal returns would be captured by the call?

Reply:
Yep -- addressed in b4c7b8878ad442a81a5fe47b6e94de287de49305 by removing the `set_cost_tracker` function and adding a `with_temporary_cost_tracker` function.